CI.md

```diff
@@ -0,0 +1,40 @@
+# Continuous Integration
+
+Each service or productive piece of software should be automatically tested, before merged to `develop`. We use [AWS CodeBuild](https://docs.aws.amazon.com/fr_fr/codebuild/) for building/testing code. For every software we have two Codebuild projects
+
+- One which is triggered by Pull Request CREATE and UPDATE event
+  - This one lints, checks formatting, builds and test the code
+- One which is triggered by Pull Request merge event
+  - This one builds and releases the build artefact (release on S3 build artifact or ECR repository)
+
+## Table of contents
+
+- [Table of contents](#table-of-contents)
+- [Guidelines](#guidelines)
+- [Build Badge](#build-badge)
+
+## Guidelines
+
+- Each Codebuild project is managed via terraform in a dedicated repository
+- `buildspec.yml` file is located within the terraform module of the Codebuild project which is in a private repository. This ensure security where a non authorized person cannot see and edit the buildspec and trigger a build on a public repository.
+- Don't use secrets/password inside the buildspec but use AWS SSM [AWS System Manager (SSM)](https://docs.aws.amazon.com/systems-manager/index.html)
+- Use build badge in the software github repository see [Build Badge](#build-badge)
+
+## Build Badge
+
+Build badge should be added to the top of the README.md file of the github project using the following template:
```

Comment:
I suggest adding a link to https://docs.aws.amazon.com/codebuild/latest/userguide/sample-build-badges.htmlI didn't know what a build badge was until I looked it up. In fact the last paragraph in this section can probably be replaced with a link to https://docs.aws.amazon.com/codebuild/latest/userguide/sample-build-badges.html#access-badges

Also, can we add a linting rule that will complain if there is no build badge link in the README of a project?

Reply:
I added the link. 
Currently we don't have README linter, which would be good for README formatting but not for such simpler case in my opinion.